### DISCUSSION TITLE
Download from archive

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#3580] [Q&A]
Not very exciting question.
The download button is not available in the archive section.

1-I import a lot of things
2-some of it I want to delete from the library but first download to put somewhere else.

The simplest and safest would be to archive first, then check in archive, download from there and delete from there exactly the current selection that was Downloaded. The usual scenario is when work images are mixed with personal images. As I cannot seem to be able to download from archive this becomes a complicated matter prone to errors.

The question: do I miss something?

**Comment by graciousgrey** [comment]
This would be possible, as there is still enough space in the context menu of the archive. You are welcome to make a feature request for this.

In the meantime you can select images in the archive ([multi-selections are also possible](https://docs.photoprism.app/user-guide/navigate/#selection-mode-and-multi-select)). Then go to Search and from there open the context menu to download the files. In any case, make sure to configure the [download settings](https://docs.photoprism.app/user-guide/settings/general/#downloads) according to your needs.

**Reply by lastzero** [reply]
The backend will likely exclude archived photos from zip archives, so when you for example download a complete album, these files are not included but only what is visible when you view the album on the UI.

**Comment by artincube** [comment]
Yes this actually work: select in archive, go back to search and selection sticks. Download, then back to archive with the selection still active and delete. I did not try as I probably saw it as confusing. I don’t see a logical reason for this. I created a feature request. It would make sense to unify the interface and add this button where it’s missing.Thank you!
